aries-site/package.json

```diff
@@ -31,6 +31,7 @@
     "test"
   ],
   "devDependencies": {
-    "babel-plugin-styled-components": "^1.10.6"
+    "babel-plugin-styled-components": "^1.10.6",
+    "testcafe": "^1.8.1"
```

Comment:
would it make sense to use `testCafe` on the core library? if so this devdependency would be a candidate to the parent package.json